docs/SRS-Volere/SRS.tex

```diff
@@ -744,15 +744,42 @@ \subsection{Adaptability Requirements}
 
 \section{Security Requirements}
 \subsection{Access Requirements}
-\lips
+\begin{enumerate} [{SR-AR}1.]
+  \item Only authorized contractors and employees should be able to access the 
+  portal for completing work on site.
+  \item Only users with managerial level access should be able to get data on 
+  contractors and employees.
+  \item Only users with administrative access should be able to make changes to 
+  site settings.
+  \item The upload and changing of certain file classes will be tied to 
+  different access levels.
+\end{enumerate}
 \subsection{Integrity Requirements}
-\lips
+\begin{enumerate} [{SR-IR}1.]
+  \item The application should not allow the addition, removal or modification 
+  of files by users without proper access and authentication.
+  \item The documents displayed should match the appropriate version available 
+  on SharePoint and/or MySDS.
+  \item The application should minimize the amount of bad or incomplete data 
+  inputted through the portal.
+\end{enumerate}
 \subsection{Privacy Requirements}
-\lips
+\begin{enumerate} [{SR-PR}1.]
+  \item Data transmission will be encrypted.
+  \item The application should protect private information in accordance with 
+  any and all government privacy laws and policies.
+\end{enumerate}
 \subsection{Audit Requirements}
-\lips
+\begin{enumerate} [{SR-AR}1.]
+  \item The application will adhere to any and all audit rules followed by City 
+  departments.
+\end{enumerate}
 \subsection{Immunity Requirements}
-\lips
+\begin{enumerate} [{SR-IR}1.]
+  \item The application will be made as secure as feasibly possible within the 
```

Comment:
Is this actually verifiable? Maybe mention some standards or saying that all components of the system should be automatically patched if there is a vulnerability found

Reply:
Ok, then could you or @kyle4908 replace it with "Security vulnerabilities and weaknesses should be patched as soon they are identified." and make the commit, I don't have the formatter installed and it fails the linter.

Reply:
nvm kyle helped me install the formatter.